packages/create-react-app/index.js

```diff
@@ -173,10 +173,14 @@ function install(packageToInstall, verbose, callback) {
 function run(root, appName, version, verbose, originalDirectory, template) {
   var packageToInstall = getInstallPackage(version);
   var packageName = getPackageName(packageToInstall);
-
   install(packageToInstall, verbose, function(code, command, args) {
     if (code !== 0) {
-      console.error(chalk.cyan(command + ' ' + args.join(' ')) + ' failed');
+      console.log();
+      console.error('Aborting installation.', chalk.cyan(command + ' ' + args.join(' ')), 'has failed.');
+      console.log();
+      console.log('Deleting', chalk.cyan(appName), 'from', chalk.cyan(originalDirectory));
+      fs.removeSync(path.join(originalDirectory, appName));
```

Comment:
Is this safe? There are cases when we allow installing into an *existing* directory if it only contains files we wouldn't clash with. I think this may actually remove user files in this case.